### ISSUE TITLE
Uncaught TypeError: Cannot read property 'toString' of null on paper-dialog

### ISSUE THREAD

**Issue by ozasadnyy** [PolymerLabs/polydev#24]
I got next error on dialog opening:

```
obj.addEventListener    @   polydev/vendor/zone-microtask.js:973
Gestures.add    @   polymer.html:774
Polymer.IronOverlayBehaviorImpl._toggleListener @   iron-overlay-behavior.html:268
(anonymous function)    @   iron-overlay-behavior.html:286
(anonymous function)    @   polymer.html:1355
arguments.(anonymous function)  @   polydev/vendor/zone-microtask.js:406
(anonymous function)    @   polydev/vendor/zone-microtask.js:98
run @   polydev/vendor/zone-microtask.js:118
zoneBoundFn @   polydev/vendor/zone-microtask.js:91
```

I was told that it's an issue related to polydev (not to polymer itself), so please take a look to it.

**Comment by mgiuffrida** [comment]
This also breaks the main elements.polymer-project.org page.

@tjsavage Is polydev still being maintained?

**Comment by tjsavage** [comment]
Yes Polydev is still maintained although the team is currently stretched. In general we recommend having Polydev running on a separate profile (i.e. not your standard browsing profile), as it does tend to break things in unexpected ways as it necessarily has to be quite invasive to pull out performance data.

**Comment by rictic** [comment]
This should be fixed in v0.0.7 which was just merged in #35 and published. Please file a new issue if you see any more problems like this.
